### PR TITLE
Update to reflect more precise terminology.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

**Overview**
We need to update our terminology to be more specific.

**Requirements**
Does the new terminology match the current best practices.

**Considered Solutions**
Other definitions were considered. 

**Recommended Solution**
Fix terminlogy.

**Considerations**
None.

### CODE REVIEW

File: modules/developers-guide/pages/concepts/data-centers.adoc [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
-The {IC} is not physical hardware that exists in any physical location. 
-Instead, the {IC} combines computing resources provided by independently-operated data centers around the world. 
+The {IC} is not physical hardware that exists in any physical location.
+Instead, the {IC} combines computing resources provided by independently-operated data centers around the world.
 
-Unlike a public or private cloud, the {IC} is not owned and operated by a single private company. 
-Instead, the {IC} is a public utility with updates and operations that are managed through an algorithmic, decentralized governance system defined in the protocol.
+Unlike a public or private cloud, the {IC} is not owned and operated by a single private company.
+Instead, the {IC} is a public utility with updates and operations that are managed through an algorithmic, decentralized proof-of-stake (POS) governance system defined in the protocol.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/data-centers.adoc:11]: remove proof of stake.
... algorithmic, decentralized governance system defined in the protocol. Any one can participate in the governance system by staking ICP token.

File: modules/developers-guide/pages/concepts/data-centers.adoc [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 Its architecture enables multiple computers to operate like one, very powerful, virtual machine.
 
-The computers that make up the virtual machine are organized into sub-networks of data centers around the globe. 
+The computers that make up the virtual machine are organized into sub-blockchain networks of data centers around the globe.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/data-centers.adoc:14]: The nodes located in data centers around the globe that make up the Internet Computer are organised in subnet blockchains that in turn connect to each other using Chain Key cryptography.

File: modules/developers-guide/pages/concepts/data-centers.adoc [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 The distributed architecture enables secure communication without firewalls or technologies that are vulnerable to attack.
-Independent data centers receive remuneration for contributing computing capacity and hosting services to support applications running on the {platform}.
+Independent data centers receive remuneration for contributing computing capacity and hosting services to support dapps running on the {IC}.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/data-centers.adoc:16]: It's really the node operates that receive remuneration. I think we should make that clearer, i.e., there are node providers that pay data centers to host their nodes.

File: modules/developers-guide/pages/concepts/data-centers.adoc [hunk 4]
```diff
@@ -30,14 +30,14 @@ In this simplified scenario:
 * Each data center has hardware supplied by multiple node providers.
 * Any single node provider might have equipment in multiple data centers.
 
-Although this example represents one subnet with nodes in multiple data centers, any of the nodes could be moved out of this subnet to form a new subnet, if needed. 
-Changes to the network topology are managed through the {IC} governance system called the **Network Nervous System** (NNS). 
+Although this example represents one blockchain subnet with nodes in multiple data centers, any of the nodes could be moved out of this blockchain subnet to form a new blockchain subnet, if needed.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/concepts/data-centers.adoc:33]: subnet blockchain  and not blockchain subnet

File: modules/developers-guide/pages/customize-projects.adoc [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
-By default, the name you use to create a new project is the name used for one data smart contract canister (`+canister_name+`) and one assets canister (`+canister_name_assets+`), and program source code is expected to be in the `+src/canister_name+` directory. Similarly, the default location for front-end source code is in the `+src/canister_name_assets/src+` directory and front-end output is located in the `+dist/canister_name_assets+` directory.
+Before you compile source code for your project using the `+dfx build+` command, you might want to check the default location for storing the source code for your dapp.
+By default, the name you use to create a new project is the name used for one data smart contract canister (`+canister_name+`) and one assets smart contract canister (`+canister_name_assets+`), and dapp source code is expected to be in the `+src/canister_name+` directory. Similarly, the default location for front-end source code is in the `+src/canister_name_assets/src+` directory and front-end output is located in the `+dist/canister_name_assets+` directory.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/customize-projects.adoc:12]: better drop smart contact here.

File: modules/developers-guide/pages/tutorials/access-control.adoc [hunk 6]
```diff
@@ -2,22 +2,23 @@
...
+To illustrate how to create and switch between user identities, this tutorial creates a simple dapp that displays a different greeting for users who are assigned to different roles.
 
 In this example, there are three named roles—`+owner+`, `+admin+`, and `+authorized+`. 
 
 * Users who are assigned an `+admin+` role see a greeting that displays `+You have a role with administrative privileges+`.
 * Users who are assigned an `+authorized+` role see a greeting that displays `+Would you like to play a game?+`.
 * Users who are not assigned one of these roles see a greeting that displays `+Nice to meet you!+`.
 
-In addition, only the user identity that initialized the canister is assigned the `+owner+` role and only the `+owner+` and `+admin+` roles can assign roles to other users.
+In addition, only the user identity that initialized the smart contract canister is assigned the `+owner+` role and only the `+owner+` and `+admin+` roles can assign roles to other users.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/tutorials/access-control.adoc:19]: please use canister smart contract or then just canister

File: modules/developers-guide/pages/tutorials/access-control.adoc [hunk 7]
```diff
@@ -68,53 +69,53 @@ To modify the default program:
...
+. Register, build, and deploy the `+access_hello+` back-end dapp by running the following command:
 +
 [source,bash]
 ----
 dfx deploy access_hello
 ----
 +
 ....
-Creating a wallet canister on the local network.
+Creating a wallet smart contract canister on the local network.
```
Comment (jancamenisch) [inline modules/developers-guide/pages/tutorials/access-control.adoc:118]: ditto

**Review by jancamenisch — CHANGES_REQUESTED** [review]
please use canister smart contract or just canister, thanks!

File: modules/developers-guide/pages/concepts/data-centers.adoc (hunk 1 above)
Comment (jplevyak) [reply modules/developers-guide/pages/concepts/data-centers.adoc]: done

File: modules/developers-guide/pages/concepts/data-centers.adoc (hunk 2 above)
Comment (jplevyak) [reply modules/developers-guide/pages/concepts/data-centers.adoc]: done

File: modules/developers-guide/pages/concepts/data-centers.adoc (hunk 3 above)
Comment (jplevyak) [reply modules/developers-guide/pages/concepts/data-centers.adoc]: done

File: modules/developers-guide/pages/concepts/data-centers.adoc (hunk 4 above)
Comment (jplevyak) [reply modules/developers-guide/pages/concepts/data-centers.adoc]: done

File: modules/developers-guide/pages/customize-projects.adoc (hunk 5 above)
Comment (jplevyak) [reply modules/developers-guide/pages/customize-projects.adoc]: done

File: modules/developers-guide/pages/tutorials/access-control.adoc (hunk 6 above)
Comment (jplevyak) [reply modules/developers-guide/pages/tutorials/access-control.adoc]: done

File: modules/developers-guide/pages/tutorials/access-control.adoc (hunk 7 above)
Comment (jplevyak) [reply modules/developers-guide/pages/tutorials/access-control.adoc]: done

**Review by jancamenisch — APPROVED** [review]

**Pull request merged**